deployment/jobs/integration-test-job.yaml

```diff
@@ -15,11 +15,11 @@ spec:
       restartPolicy: Never
       containers:
       - name: entitytester
-        image: quay.io/n1analytics/entity-app
+        image: data61/anonlink-app:v1.12.0
```

Comment:
wouldn't it be better to link to `latest` or `stable` or something similar. We don't want to update the deployment files with every release.

Reply:
Sure, happy to use `latest`. It is mostly there as an example, and pinning exact versions of what runs is considered best practice.

Reply:
I'll remove the log level setting from here too - it is just for the logging level during tests which pytest overrides anyway.